lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -207,6 +219,8 @@ def setup(self):
             self._loop.call_later(self._redis_conn_retry_secs, self.setup)
 
     def _setupInternal(self):
+        if self._initialized:
+            return
```

Comment:
Maybe debug log that we're already initialized

Reply:
I do not sure if this will help.